nav2_bringup/launch/nav2_bringup_launch.py

```diff
@@ -31,9 +31,8 @@ def generate_launch_description():
         launch_ros.actions.Node(
             package='nav2_amcl',
             node_executable='amcl',
-            node_name='amcl',
```

Comment:
I think this line is the only one that matters for working around ros2/rcl#375. The change to how parameters are passed doesn't look to be necessary.

Reply:
Ah never mind, I didn't realize launch required `node_name` when using this form of paramters

`[ERROR] [launch.LaunchService]: Caught exception in launch (see debug for traceback): If a dictionary of parameters is specified, the node name must also be specified. See https://github.com/ros2/launch/issues/139
```